flask_server.py

```diff
@@ -14,27 +15,63 @@ def setLogFileList():
     with open("run_history.log", "r") as o:
         return [log_line for log_line in o.readlines()]
 
+def savePaths(cookie_path, folder_path):
+    paths_dict = {'cookie_path' : cookie_path, 'folder_path' : folder_path}
+    with open('paths.json', 'w') as f:
+        json.dump(paths_dict, f)
 
-@app.route("/")
+def setUpdatedPaths():
+    with open('paths.json', 'r') as f:
+        paths_dict = json.load(f)
+
+    # Returns cookie_path, folder_path
+
+    return paths_dict["cookie_path"], paths_dict["folder_path"]
+
+################### ROUTES #################
+
+@app.route("/", methods=['GET','POST'])
 def index():
     log_line = setLogFileList()
-    return render_template("index.html", count_i = 0, count_v = 0, log_line = log_line)
-
-	
-@app.route("/scrape/", methods=['POST'])
 
-def scrape():
     if request.method == "POST":
         amountScraped = int(request.form["amountToScrape"])
-        count_i, count_v = scrape_from_web(COOKIE_PATH, amountScraped)
-
+        count_i, count_v = scrape_from_web(cookie_path, folder_path, amountScraped)
         log_line = setLogFileList()
-        return render_template('index.html', count_i = count_i, count_v = count_v, log_line = log_line)
+        return render_template('index.html', count_i = count_i, count_v = count_v, log_line = log_line, error = "0")
 
+    return render_template("index.html", count_i = 0, count_v = 0, log_line = log_line)
 
 	
+@app.route("/settings/", methods=['GET','POST'])
+def settings():
+    global cookie_path, folder_path
```

Comment:
Avoid global. 

Use:
def getSettings()
and
def setSettings()

Remove all mentions of paths